user-interface/src/data-verification/ConsolidationOrderAccordion.tsx

```diff
@@ -81,21 +84,46 @@ export function ConsolidationOrderAccordion(props: ConsolidationOrderAccordionPr
     throw new Error('Function not implemented.');
   }
 
+  async function handleOnExpand() {
+    if (props.onExpand) {
+      props.onExpand(`order-list-${order.id}`);
+    }
+    if (!isAssignmentLoaded) {
+      for (const bCase of order.childCases) {
+        try {
+          const assignmentsResponse = await api.get(`/case-assignments/${bCase.caseId}`);
+          bCase.attorneyAssignments = (assignmentsResponse as CaseAssignmentResponseData).body;
+        } catch {
+          // The case assignments are not critical to perform the consolidation. Catch any error
+          // and don't set the attorney assignment for this specific case.
+        }
+      }
+      // Ensure the loaded assignments are stored in state on the order and not overridden
+      // should the parent data consolidation screen refresh the accordion.
+      setOrder({ ...order });
+      setIsAssignmentLoaded(true);
```

Comment:
Could we write a named function that would indicate why we are doing these things rather than the comments?